### ISSUE TITLE
update device reboot crontab

### ISSUE THREAD

**Issue by alihanks** [bearing/dosenet-raspberrypi#73]
change crontab to call dosenet.sh finish
make system update script to implement this change on all stations

**Comment by bplimley** [comment]
Waiting on #96

**Comment by bplimley** [comment]
Actually, this needs to be more complicated, because if `manager.py` isn't running, then `dosenet.sh finish` won't do anything, and the device will be stuck...

**Comment by alihanks** [comment]
Good point.
